src/nupic/encoders/base.py

```diff
@@ -494,53 +513,59 @@ def decode(self, encoded, parentFieldName=''):
-    method.
+    that would have generated a specific encoded output, use the
+    :meth:`.topDownCompute` method.
 
     If you want to pretty print the return value from this method, use the
-    decodedToStr() method.
+    :meth:`.decodedToStr` method.
 
     :param encoded:      The encoded output that you want decode
     :param parentFieldName: The name of the encoder which is our parent. This name
            is prefixed to each of the field names within this encoder to form the
            keys of the dict() in the retval.
 
-    :return: tuple(fieldsDict, fieldOrder) (see below for details)
+    :return: tuple(``fieldsDict``, ``fieldOrder``)
 
-    fieldsDict is a dict() where the keys represent field names
-    (only 1 if this is a simple encoder, > 1 if this is a multi
-    or date encoder) and the values are the result of decoding each
-    field. If there are  no bits in encoded that would have been
-    generated by a field, it won't be present in the dict. The
-    key of each entry in the dict is formed by joining the passed in
-    parentFieldName with the child encoder name using a '.'.
+              ``fieldsDict`` is a dict() where the keys represent field names
+              (only 1 if this is a simple encoder, > 1 if this is a multi
+              or date encoder) and the values are the result of decoding each
+              field. If there are  no bits in encoded that would have been
+              generated by a field, it won't be present in the dict. The
+              key of each entry in the dict is formed by joining the passed in
+              parentFieldName with the child encoder name using a '.'.
 
-    Each 'value' in fieldsDict consists of (ranges, desc), where
-    ranges is a list of one or more (minVal, maxVal) ranges of
-    input that would generate bits in the encoded output and 'desc'
-    is a pretty print description of the ranges. For encoders like
-    the category encoder, the 'desc' will contain the category
-    names that correspond to the scalar values included in the
-    ranges.
+              Each 'value' in ``fieldsDict`` consists of (ranges, desc), where
+              ranges is a list of one or more (minVal, maxVal) ranges of
+              input that would generate bits in the encoded output and 'desc'
+              is a pretty print description of the ranges. For encoders like
+              the category encoder, the 'desc' will contain the category
+              names that correspond to the scalar values included in the
+              ranges.
 
-    The fieldOrder is a list of the keys from fieldsDict, in the
-    same order as the fields appear in the encoded output.
+              ``fieldOrder`` is a list of the keys from ``fieldsDict``, in the
+              same order as the fields appear in the encoded output.
 
-    TODO: when we switch to Python 2.7 or 3.x, use OrderedDict
+              TODO: when we switch to Python 2.7 or 3.x, use OrderedDict
```

Comment:
There is a TODO extension that you might be interested in.

Ref 1: https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#todo-extension
Ref 2: http://www.sphinx-doc.org/en/1.0.8/ext/todo.html